Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -1357,21 +1322,31 @@ static std::vector<LoadModelResult> loadModelAnyThreadPart(
     // Show the default scene
     const CesiumGltf::Scene& defaultScene = model.scenes[model.scene];
     for (int nodeId : defaultScene.nodes) {
-      loadNode(result, model, model.nodes[nodeId], rootTransform, options);
+      CreateNodeOptions nodeOptions = options;
+      nodeOptions.pNode = &model.nodes[nodeId];
```

Comment:
I think it'd be a little clearer and safer to construct the CreateNodeOptions from the two things explicitly, like:

```
CreateNodeOptions nodeOptions(options, &model.nodes[nodeId]);
```